src/main/webapp/assets/vendors/AdminLTE-2.4.2/css/AdminLTE.css

```diff
@@ -172,13 +172,6 @@ h6,
 a {
   color: #337ab7;
 }
-a:hover,
-a:active,
-a:focus {
-  outline: none;
-  text-decoration: none;
-  color: #23527c;
-}
```

Comment:
What for?

BTW, do not modify this css file directly as it is from third party library.  Anything you modified would be lost when AdminLTE upgraded.

Reply:
> What for?
> 

Sorry, you clarified here
>Hiding focus indicators is unhelpful for people navigating by keyboard.
Show them instead.

Reply:
Hmm, why  did you remove `color: #23527c;`? Screenshots before and after would help.

Reply:
How else do I modify this CSS? It seems that the way AdminLTE indicates focus is by slightly darkening links. I can't even see the difference. So this commit replaces it with classic browser focus stuff. You can compare it by tabbing through the GitBucket demo and https://git.xogium.me/